docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -242,3 +242,58 @@ As you can see, we get different groups of hosts.
 
 In case a filter parameter is not defined, the plugin supposes all values possible are wanted.
 This means that for each tag that exists on your Scaleway compute nodes, a group based on each tag will be created.
+
+Scaleway S3 object storage
+==========================
+
+Object Storage allows you to store any kind of objects (documents, images, videos, etc.).
+As the Scaleway API is S3 compatible, Ansible supports it natively through the modules: :ref:`s3_bucket_module_module`, :ref:`aws_s3_module`.
+
+You can find many examples in `./test/legacy/roles/scaleway_s3`
+
+.. code-block:: yaml
+
+    - hosts: myserver
+      vars:
+        scaleway_region: nl-ams
+        s3_url: https://s3.nl-ams.scw.cloud
+      environment:
+        # AWS_ACCESS_KEY matches your scaleway organization id available at https://cloud.scaleway.com/#/account
+        AWS_ACCESS_KEY: 00000000-1111-2222-3333-444444444444
+        # AWS_SECRET_KEY matches a secret token that you can retrieve at https://cloud.scaleway.com/#/credentials
+        AWS_SECRET_KEY: 00000000-1111-2222-3333-444444444444
```

Comment:
I would use a different value for `AWS_SECRET_KEY` that the one used for `AWS_ACCESS_KEY`.

Reply:
+1 - otherwise it's easy to think the two variables should be the same, or that it's a typo, etc.